sample/AppHttpSample/AppHttpSample.csproj

```diff
@@ -0,0 +1,26 @@
+<Project Sdk="Microsoft.NET.Sdk.Web">
+
+    <PropertyGroup>
+        <TargetFramework>net6.0</TargetFramework>
+        <Nullable>enable</Nullable>
+        <ImplicitUsings>enable</ImplicitUsings>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\..\src\EasyAbp.Abp.EventBus.Boxes.Dtm.EntityFrameworkCore\EasyAbp.Abp.EventBus.Boxes.Dtm.EntityFrameworkCore.csproj" />
+      <ProjectReference Include="..\..\src\EasyAbp.Abp.EventBus.Boxes.Dtm.Http\EasyAbp.Abp.EventBus.Boxes.Dtm.Http.csproj" />
+      <ProjectReference Include="..\..\src\EasyAbp.Abp.EventBus.Boxes.Dtm\EasyAbp.Abp.EventBus.Boxes.Dtm.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Volo.Abp.AspNetCore.Mvc" Version="5.3.1" />
+      <PackageReference Include="Volo.Abp.Autofac" Version="5.3.1" />
+      <PackageReference Include="Volo.Abp.Caching" Version="5.3.1" />
+      <PackageReference Include="Volo.Abp.EventBus.RabbitMQ" Version="5.3.0" />
+        <PackageReference Include="Serilog.AspNetCore" Version="5.0.0" />
+        <PackageReference Include="Serilog.Sinks.Async" Version="1.5.0" />
+        <PackageReference Include="Volo.Abp.EntityFrameworkCore.PostgreSql" Version="5.3.0" />
```

Comment:
1. Please use LocalDB for a more straightforward run.
2. Don't forget `[DependsOn]`.